src/MoneyMarket.java

```diff
@@ -49,27 +55,44 @@ public void updateLoyalStatus() {
         isLoyal = balance >= MIN_AMOUNT;
     }
 
+
     /**
      * method to make a withDrawl
      * @param amount
      */
+    @Override
```

Comment:
this does the same except it increases withdraws? why not just call the superclass's method then add this as extra code rather than rewriting the whole thing which includes a lot of copy pasted code?
something like this:

super.makeWithdraw(amount);
withdraaws++;
if ...